cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { BreweryParameters, Login } from '../../commons/actions/brewery';
+import { ScenarioParameters, ScenarioManager } from '../../commons/actions/generic';
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+import { Scenarios } from '../../commons/actions';
+import { ScenarioSelector } from '../../commons/actions/generic/ScenarioSelector';
+import utils from '../../commons/TestUtils';
+
+const SCENARIO_DATASET = DATASET.BREWERY_ADT;
+const SCENARIO_RUN_TEMPLATE = RUN_TEMPLATE.BASIC_TYPES;
```

Comment:
why do we need to assign the variable to another variable used only once ? :thinking:

Reply:
It is inspired by what has already been made in other tests, by exemple in RenameScenario.
but indeed, it is not an absolute requirement.